src/features/authentication/authSlice.tsx

```diff
@@ -2,35 +2,78 @@ import { createSlice } from '@reduxjs/toolkit';
 import { PayloadAction } from '@reduxjs/toolkit';
 
 interface UserDetails {
-  userName: string;
+  username: string;
   isLoggedIn: boolean;
+  isAdmin: boolean;
```

Comment:
In user story, I think only manager need login system. So, I think isLoggedin attribute can act as isAdmin as well.

Reply:
I am not quite sure as of yet. There is also mentioned registered user story. I intend to make a login interface for users as well.